Documentation/Usage.md

```diff
@@ -517,6 +518,60 @@ Since package pin information is *not* inherited across dependencies, our recomm
     
     When automatic pinning is enabled, package update act as if `--repin` was specified.
 
+## Swift Tools Version
+
+The tools version declares the minimum version of the Swift tools required to
+use the package, determines what version of the PackageDescription API should
+be used in the Package.swift manifest, and determines which Swift language
+compatibility version should be used to parse the Package.swift manifest.
+
+When resolving package dependencies, if the version of a dependency that would
+normally be chosen specifies a Swift tools version which is greater than the
+version in use, that version of the dependency will be considered ineligible
+and dependency resolution will continue with evaluating the next-best version.
+If no version of a dependency (which otherwise meets the version requirements
+from the package dependency graph) supports the version of the Swift tools in
+use, a dependency resolution error will result.
+
+### Swift Tools Version Specification
+
+The Swift tools version is specified by a special comment in the first line of
+the Package.swift manifest. To specify a tools version, a Package.swift file
+must begin with the string `// swift-tools-version:`, followed by a version
+number specifier.
```

Comment:
it might be worth saying that this isn't required, although enabling it supports working on packages with different versions of swift.